File: Assignment-02-May/countvowels.js
"use strict";
exports.__esModule = true;
var readline = require("readline");
var countvowels = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});
countvowels.question("enter the String :", function (answer) {
    var count = 0;
    var newarr = (answer).split('');
    for (var i = 0; i < newarr.length; i++) {
        if (newarr[i] == 'a' || newarr[i] == 'e' || newarr[i] == 'i' || newarr[i] == 'o' || newarr[i] == 'u') {
            count++;
        }
    }
    console.log("The count of vowels is " + "" + count);
    countvowels.close();
});

File: Java_Assignments_09_May/CountVowels.java
import java.util.Scanner;
class CountVowels{
	public static void main(String[] args) {
		int count = 0;
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter the string to count vowels in string");
		String str = sc.nextLine();
		for(int i =0; i< str.length(); i++){
			if(str.charAt(i) == 'a' || str.charAt(i) == 'e'|| str.charAt(i) == 'i' || str.charAt(i) == 'o' || str.charAt(i) == 'u'){
				count ++;
			}
			
		}
		System.out.println("count of vowels" +count);
	}
}